src/Mailgun/Api/Pagination.php

```diff
@@ -10,6 +10,7 @@
 namespace Mailgun\Api;
 
 use Mailgun\Assert;
+use Mailgun\Resource\Api\mixed;
```

Comment:
👎 this is not a valid resource that we can import. If we can use more than one based on "criteria" then I suggest to add `use Mailgun\Resource\Api;` and other annotations and/or type hint can use `Api\...`